loopy/tools.py

```diff
@@ -732,6 +732,11 @@ def _kernel_to_python(kernel, is_entrypoint=False, var_name="kernel"):
             option += ("dep="+":".join(insn.depends_on)+", ")
         if insn.tags:
             option += ("tags="+":".join(insn.tags)+", ")
+        if insn.within_inames:
```

Comment:
```suggestion
        if insn.within_inames is not None:
```
? Even an empty `within_inames` may be worth recording.